### PR TITLE
Add correct pattern for responsive classes

### LINKED ISSUE #9317: PrimeFaces CLI: grid migration issue
I was migrating an old project from grid css to primeflex and I noticed responsive classe are not converted.
The grid.css pattern for responsive classes, as you can see on [the showcase](https://www.primefaces.org/showcase-v8/ui/panel/grid.xhtml) or [the documentation](https://primefaces.github.io/primefaces/8_0/#/components/gridcss), is ui-(xl|lg|md|sm)-_n_,
while the migration tool looks only for ui-g-(xl|lg|md|sm)-_n_:
https://github.com/primefaces/primefaces/blob/9c257a48b28464436e49c4bc782b2233ba23925e/primefaces-cli/src/main/java/org/primefaces/cli/migration/primeflex/GridCssMigration.java#L47

PR coming

### PR DESCRIPTION
**Pull request by NicolaIsotta**
fixes #9317

### CODE REVIEW

File: primefaces-cli/src/main/java/org/primefaces/cli/migration/primeflex/GridCssMigration.java [hunk 1]
```diff
@@ -45,6 +45,7 @@ protected void initReplaceRegEx() {
 
         // grid
         replaceRegex.put("ui-g-(xl|lg|md|sm)-([0-9]+?)", "p-(xl|lg|md|sm)-([0-9]+?)");
```
Comment (jepsar) [inline primefaces-cli/src/main/java/org/primefaces/cli/migration/primeflex/GridCssMigration.java]: @melloware shouldn't groups be referenced in the replacement here? I would expect `"p-$1-$2"`.

Comment (melloware) [reply primefaces-cli/src/main/java/org/primefaces/cli/migration/primeflex/GridCssMigration.java]: Yes this looks wrong it should be $ like jepsar says

**Review by melloware — CHANGES_REQUESTED** [review]

File: primefaces-cli/src/main/java/org/primefaces/cli/migration/primeflex/GridCssMigration.java (hunk 1 above)
Comment (jepsar) [reply primefaces-cli/src/main/java/org/primefaces/cli/migration/primeflex/GridCssMigration.java]: In that case we don't have a unit test for that replacement, or it's wrong as well

**Review by melloware — APPROVED** [review]

**Comment by NicolaIsotta** [comment]
fixed and tested locally with the showcase page, everything works fine.

**Pull request merged**